src/Input/Silk.NET.Input.Desktop/GlfwInputContext.cs

```diff
@@ -0,0 +1,81 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Silk.NET.GLFW;
+using Silk.NET.Input.Common;
+using Silk.NET.Input.Desktop.Collections;
+using Silk.NET.Windowing.Common;
+
+namespace Silk.NET.Input.Desktop
+{
+    public class GlfwInputContext : IInputContext
+    {
+        internal List<GlfwGamepad> _gamepads;
+        internal List<GlfwJoystick> _joysticks;
+        internal IKeyboard _keyboard;
+        internal IMouse _mouse;
+        internal IWindow _window;

```

Comment:
I think we should store the keyboard, mouse and window as the glfw version, that way we are sure they are the correct type. And it makes more sense as this is a glfw context